cli/sawtooth_cli/main.py

```diff
@@ -25,14 +25,11 @@
 
 from sawtooth_cli.exceptions import CliException
 
-from sawtooth_cli.admin import add_admin_parser
-from sawtooth_cli.admin import do_admin
-from sawtooth_cli.keygen import add_keygen_parser
-from sawtooth_cli.keygen import do_keygen
-from sawtooth_cli.config import add_config_parser
-from sawtooth_cli.config import do_config
-from sawtooth_cli.block import add_block_parser
-from sawtooth_cli.block import do_block
+from sawtooth_cli.admin import add_admin_parser, do_admin
+from sawtooth_cli.keygen import add_keygen_parser, do_keygen
+from sawtooth_cli.config import add_config_parser, do_config
+from sawtooth_cli.block import add_block_parser, do_block
+from sawtooth_cli.state import add_state_parser, do_state
```

Comment:
Prefer the original format (one import per line) as it is more diff-friendly and thus easier to resolve merge conflicts.